packages/web-app-files/src/views/Favorites.vue

```diff
@@ -42,13 +42,7 @@
           </template>
           <template #footer>
             <pagination :pages="paginationPages" :current-page="paginationPage" />
-            <list-info
-              v-if="paginatedResources.length > 0"
-              class="oc-width-1-1 oc-my-s"
-              :files="totalResourcesCount.files"
-              :folders="totalResourcesCount.folders"
-              :size="totalResourcesSize"
```

Comment:
I guess `totalResourcesCount` and `totalResourcesSize` can now be removed as well here? Same goes for the other views.

Reply:
You've adjusted the `Favorites` view, but the other views are still missing. Don't know if it can be removed in all of them, but in some at least.

Reply:
i hope that all now